events.html

```diff
@@ -0,0 +1,106 @@
+        <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet">
+
+    </head>
+    <body>
+
+        <div class="container fluid">
+        <!-- Navigation -->
+            <header>
+                <nav class="navbar navbar-expand-lg navbar-light bg-light fixed-top">
+                    <a class="navbar-brand" href="index.html#">
+                        <img src="/img/cloud-custodian.svg" width="30" height="30" class="d-inline-block align-top" alt="Cloud Custodian logo">
+                        Cloud Custodian
+                    </a>
+                    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarColor03" aria-controls="navbarColor03" aria-expanded="false" aria-label="Toggle navigation">
+                        <span class="navbar-toggler-icon"></span>
+                    </button>
+
+                    <div class="collapse navbar-collapse" id="navbarColor03">
+                        <ul class="navbar-nav ml-auto">
+                            <li class="nav-item">
+                                <a class="nav-link" href="docs/index.html ">
+                                    <i class="fas fa-book fa-fw "></i>
+                                    Docs
+                                </a>
+                            </li>
+                            <li class="nav-item">
+                                <a class="nav-link" href="https://pypi.python.org/pypi/c7n ">
+                                    <i class="fas fa-cloud-download-alt fa-fw "></i>
+                                    Install
+                                </a>
+                            </li>
+                            <li class="nav-item">
+                                <a class="nav-link" href="https://github.com/cloud-custodian/cloud-custodian ">
+                                    <i class="fab fa-sm fa-github"></i>
+                                    GitHub
+                                </a>
+                            </li>
+                            <li class="nav-item">
+                                <a class="nav-link" href="https://gitter.im/cloud-custodian/cloud-custodian ">
+                                    <i class="fab fa-sm fa-gitter"></i>
+                                    Community
+                                </a>
+                            </li>
+                            <li class="nav-item">
+                                <a class="nav-link" href="events.html">
+                                    <i class="fa fa-calendar"></i>
+                                    Calendar
+                                </a>
+                            </li>
+                        </ul>
+                    </div>
+                </nav>
+            </header>
+
+            <main role="main" style="padding-top: 5em">
```

Comment:
```suggestion
            <main style="padding-top: 5em">
```

main is already main, a role stating as such is redundant